tests/data/validation/enum/enum.go

```diff
@@ -124,6 +144,16 @@ func (j *EnumMyIntegerTypedEnum) UnmarshalJSON(b []byte) error {
 	return nil
 }
 
+// Verify checks all fields on the struct match the schema.
+func (plain EnumMyIntegerTypedEnum) Verify() error {
+	for _, expected := range enumValues_EnumMyIntegerTypedEnum {
+		if reflect.DeepEqual(v, expected) {
```

Comment:
where does this `v` come from? I don't see it declared anywhere, are you sure it's valid?